lemur/plugins/lemur_sectigo/plugin.py

```diff
@@ -119,10 +119,11 @@ def _collect_certificate(cert_id, ssl_client):
         current_app.logger.info({"message": "Collecting certificate from Sectigo..."})
         cert_pem = ssl_client.collect(cert_id=cert_id, cert_format="pem")
         parts = pem.parse(cert_pem.encode("utf-8"))
-        ca_bundle = [str(c) for c in parts[:-1]]
-        ca_bundle.reverse()
-        ca_bundle = "".join(ca_bundle)
-        issued_cert = str(parts[-1])
+        parts = [str(c) for c in parts]
+        issued_cert = parts.pop()
+        parts.reverse()
+        # Don't include any self-signed (anchor/root CA) certs in the bundle
+        ca_bundle = "".join([c for c in parts if parse_certificate(c).issuer != parse_certificate(c).subject])
```

Comment:
It's good this defensive check is in place otherwise it could inadvertently remove a valid intermediate certificate we would need if the API changes in the future or doesn't return a root certificate in some cases.